### PR TITLE
Added AllMarketTickers channel

### PR DESCRIPTION
**Pull request by tonyskulk**

### CODE REVIEW

**Comment by xezon** [comment]
Very nice.

File: src/main/java/com/binance/api/client/domain/event/AllMarketTickersEvent.java [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+  public long getTotalNumberOfTrades() {
+    return totalNumberOfTrades;
+  }
+
+  public void setTotalNumberOfTrades(long totalNumberOfTrades) {
+    this.totalNumberOfTrades = totalNumberOfTrades;
+  }
+
+  @Override
+  public String toString() {
```
Comment (joaopsilva) [inline src/main/java/com/binance/api/client/domain/event/AllMarketTickersEvent.java]: Could you please replace this to use the `ToStringBuilder` that is used in other transfer objects? Thanks.

**Comment by tonyskulk** [comment]
Switched to ToStringBuilder and made a very simple self-explaining example

**Comment by joaopsilva** [comment]
Hi @tonyskulk , could you please have a look at the issue reported in https://github.com/binance-exchange/binance-java-api/issues/72? Thanks.

**Comment by mjwchapman** [comment]
Would be useful to update the example so that it executes the problematic code path, e.g.
```
        webSocketClient.onAllMarketTickersEvent(eventList -> {
            try {
                for (AllMarketTickersEvent event : eventList) {
                    System.out.println(event);
                }
            } catch (Exception e) {
                e.printStackTrace();
            }
        });
```

**Pull request merged**